### ISSUE TITLE
Build failed for ReactiveMapKit with Carthage

### ISSUE THREAD

**Issue by sebastianvarela** [ReactiveCocoa/ReactiveCocoa#3627]
Hi!
Since a few days ago I'm having problems compiling ReactiveMapKit with Carthage:
```
#carthage update --platform iOS --configuration Debug --cache-builds
*** Fetching ReactiveCocoa
*** Fetching ReactiveSwift
*** Fetching Result
*** Checking out ReactiveCocoa at "8.0.1"
*** Checking out ReactiveSwift at "4.0.0"
*** Checking out Result at "4.0.0"
*** xcodebuild output can be found in /var/folders/p5/7pkbst7151d3y7ff8w0bt7j00000gp/T/carthage-xcodebuild.ZAzig8.log
*** Valid cache found for Result, skipping build
*** Valid cache found for ReactiveSwift, skipping build
*** Invalid cache found for ReactiveCocoa, rebuilding with all downstream dependencies
*** Building scheme "ReactiveCocoa-iOS" in ReactiveCocoa.xcworkspace
*** Building scheme "ReactiveMapKit" in ReactiveCocoa.xcworkspace
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -workspace /Users/sebastianvarela/repos/personal/github/Power/Carthage/Checkouts/ReactiveCocoa/ReactiveCocoa.xcworkspace -scheme ReactiveMapKit -configuration Debug -derivedDataPath /Users/sebastianvarela/Library/Caches/org.carthage.CarthageKit/DerivedData/10.0_10A255/ReactiveCocoa/8.0.1 -sdk iphonesimulator -destination platform=iOS\ Simulator,id=48C5D680-1410-450E-B173-AE771AFC46F0 -destination-timeout 3 ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES build (launched in /Users/sebastianvarela/repos/personal/github/Power/Carthage/Checkouts/ReactiveCocoa)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/p5/7pkbst7151d3y7ff8w0bt7j00000gp/T/carthage-xcodebuild.ZAzig8.log
```

```
▸ Compiling WKInterfaceLabel.swift

❌  /Users/sebastianvarela/repos/personal/github/Power/Carthage/Checkouts/ReactiveCocoa/ReactiveCocoa/WatchKit/WKInterfaceLabel.swift:4:32: 'WKInterfaceLabel' is only available on iOS application extension 8.2 or newer

extension Reactive where Base: WKInterfaceLabel {
                                ^



❌  /Users/sebastianvarela/repos/personal/github/Power/Carthage/Checkouts/ReactiveCocoa/ReactiveCocoa/WatchKit/WKInterfaceLabel.swift:4:32: 'WKInterfaceLabel' is only available on iOS application extension 8.2 or newer

extension Reactive where Base: WKInterfaceLabel {
^



❌  /Users/sebastianvarela/repos/personal/github/Power/Carthage/Checkouts/ReactiveCocoa/ReactiveCocoa/WatchKit/WKInterfaceLabel.swift:4:32: 'WKInterfaceLabel' is only available on iOS application extension 8.2 or newer

extension Reactive where Base: WKInterfaceLabel {
^



❌  /Users/sebastianvarela/repos/personal/github/Power/Carthage/Checkouts/ReactiveCocoa/ReactiveCocoa/WatchKit/WKInterfaceButton.swift:4:32: 'WKInterfaceButton' is only available on iOS application extension 8.2 or newer

extension Reactive where Base: WKInterfaceButton {
^



❌  /Users/sebastianvarela/repos/personal/github/Power/Carthage/Checkouts/ReactiveCocoa/ReactiveCocoa/WatchKit/WKInterfaceButton.swift:4:32: 'WKInterfaceButton' is only available on iOS application extension 8.2 or newer

extension Reactive where Base: WKInterfaceButton {
^



❌  /Users/sebastianvarela/repos/personal/github/Power/Carthage/Checkouts/ReactiveCocoa/ReactiveCocoa/WatchKit/WKInterfaceButton.swift:4:32: 'WKInterfaceButton' is only available on iOS application extension 8.2 or newer
```

This happens in my two workstation and in my travis-ci: https://travis-ci.org/sebastianvarela/Power/builds/440984013

Thanks!

**Comment by andersio** [comment]
Hi, this should have been resolved by #3625 and we will release 8.0.2 for this ASAP. Sorry for any inconvenience caused.

**Comment by andersio** [comment]
8.0.2 has been released. If the issue persists, please do reopen the issue. 🙂
